src/requirements/data/majors/aem.ts

```diff
@@ -20,16 +20,295 @@ const aemRequirements: readonly CollegeOrMajorRequirement[] = [
     fulfilledBy: 'credits',
     perSlotMinCount: [55],
   },
+  // TODO: only supports one concentration
   {
     name: 'AEM Concentration Requirement',
+    // TODO: counting is temp courses. should eventually be credits
```

Comment:
The code that mandates only one slot for credits is here: https://github.com/cornell-dti/course-plan/blob/54a7e3e00bfee1b15edcec5f41e4ed5fcc6aede2/src/components/Requirements/SubRequirement.vue#L257-L269

I think the code can be easily extended to support multiple slots to support your use case, but the end result might be a little confusing for end users.

e.g. suppose we have 2 credits slots with `perSlotMinCount  = [6, 6]`. Suppose for each slot we have only one existing course of 3 credit. Then the frontend might render something like

```text
-- Add Course 1 ----------------------------
| Course foo for slot 1, already fulfilled |
--------------------------------------------

-- Add Course 2 ----------------------------
| unfulfilled course for slot 1            |
--------------------------------------------

-- Add Course 3 ----------------------------
| Course foo for slot 1, already fulfilled |
--------------------------------------------

-- Add Course 4 ----------------------------
| unfulfilled course for slot 1            |
--------------------------------------------
```

so we should probably do this for post launch after we have slot names?

Reply:
that sounds like a good idea, ty sam!